atomkraft/cli/__init__.py

```diff
@@ -25,3 +28,27 @@ def init(path: Path):
 @app.command()
 def smoke_test():
     pytest.main([".atomkraft/smoke_tests"])
+
+
+# run with:
+# poetry run atomkraft reactor --actions-list=act1,act2,act3 --variables-list=x,y,z --reactor-stub-file=path/to/the/file
+# or
+# poetry run atomkraft reactor --actions-list="act1, act2, act3" --variables-list="x, y, z" --reactor-stub-file="path/to/the/file"
+
+
+@app.command("reactor")
+def reactor_command(
+    variables_list: str = typer.Option(
+        ..., help="A list of state variables to include into reactors as parameters"
+    ),
+    actions_list: Optional[str] = typer.Option(
+        ..., help="A list of actions for which to create reactor stubs."
+    ),
+    reactor_stub_file: str = typer.Option(
+        os.path.abspath("reactors/reactor.py"),
+        help="A path where to create the reactors file.",
+    ),
+):
+    actions = [act.strip() for act in actions_list.split(",")]
+    variables = [var.strip() for var in variables_list.split(",")]
+    reactor.generate_reactor(actions, variables, reactor_stub_file)
```

Comment:
How about simplifying this a bit?

```suggestion
    actions: str = typer.Argument(
        ..., help="trace actions for which to create reactor stub functions.",
        show_default=False
    ),
    variables: str = typer.Argument(
        ..., help="state variables to use as parameters for reactor stub actions",
        show_default=False
    ),
    path: typer.FileTextWrite = typer.Option(
        os.path.abspath("reactors/reactor.py"),
        help="path where to create the reactor stub",
    ),
):
    '''
    Generate reactor stub used to interpret test traces
    '''
    actions = [act.strip() for act in actions.split(",")]
    variables = [var.strip() for var in variables.split(",")]
    reactor.generate_reactor(actions, variables, path.name)
```